wordpress-mobile/WordPress-Android
# Issue/12263 updated post creation qs task

Fixes #12263

This PR updates the flow for Publish Post Quick Start Task to accommodate IA changes.

Quick Start is relatively flexible in ways it could be used, but it also means it's hard to have a lot of universal functionality for it, and there is a lot of special cases. It can totally use refactoring to accommodate potential new use cases, but it's outside of the scope of this project.

[![Image from Gyazo](https://i.gyazo.com/c05103e93ef248aa04f8c58f028e02e4.png)](https://gyazo.com/c05103e93ef248aa04f8c58f028e02e4)

To test:
- Create a new site, and start a Quick Start.
- Navigate to Grow Your Audience category, and select `Publish a post ` task.
- Make sure content of Snackbar is updated, and the focus point is displayed above FAB.
- Rotate the device and make sure the focus point is still visible.
- Tap on the FAB.
- Confirm that bottom dialog is shown and focus point is displayed on `Blog post` row.
- Rotate the device and make sure the focus point is still visible.
- Tap on `Blog post` row, which will take you to the Editor.
- Navigate back from Editor and confirm that the task was completed.

PR submission checklist:
